crates/bevy_ui/src/entity.rs

```diff
@@ -39,6 +42,38 @@ pub struct NodeBundle {
     pub computed_visibility: ComputedVisibility,
 }
 
+impl NodeBundle {
+    pub fn new(style: Style, color: impl Into<UiColor>) -> Self {
+        Self {
+            style,
+            color: color.into(),
+            ..default()
+        }
+    }
+
+    /// Returns a node without a background color and that will let interactions go through them
+    /// A transparent node is like a <div> element in HTML
+    pub fn transparent() -> Self {
```

Comment:
Other suggestions:

- pass_through
- empty
- container
- shell

Reply:
I don't like `empty` because it's used as a container and it's not really useful if it was actually empty.

I live container since it's pretty much what it is

Reply:
It's one less character too 😉